lua/lspconfig/configs.lua

```diff
@@ -136,7 +137,7 @@ function configs.__newindex(t, config_name, config_def)
         end
 
         if root_dir then
-          api.nvim_create_autocmd('BufReadPost', {
+          buf_read_post_autocmd_id = api.nvim_create_autocmd('BufReadPost', {
             pattern = fn.fnameescape(root_dir) .. '/*',
             callback = function(arg)
               M.manager:try_add_wrapper(arg.buf, root_dir)
```

Comment:
Can the callback check if the client was force stopped? Then it can return `true`, which will automatically remove the autocmd.

```suggestion
              if client.is_stopped -- psuedocode
                return true
              end
              M.manager:try_add_wrapper(arg.buf, root_dir)
```

Reply:
Nice, with this addressed, other two comments should be also good.

Thanks, updated.